

try:
    raise Exception("should be between values")

except:
    if help:
        print(help)


